ruby-training/trainees/shareni/test_katas/05/sum_array/array_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rspec'
+require_relative 'new_array'
+
+describe "Array" do
```

Comment:
Remember, "describe" block descriptions should match class names and/or method names... In this case your class is called `NewArray` but you're describing `Array`. Next you describe the method `#sum` yet none of the class' method have that name. Finally, you're only testing one of your two public methods (although you have written extensive tests for it, which is really good).